analyses/molecular-subtyping-pathology/pathology_free_text-subtyping-meningioma.Rmd

```diff
@@ -100,16 +103,24 @@ meningioma_df <- histologies_df %>%
       # This will be true when none of the conditions (e.g., strings are 
       # detected) above
       TRUE ~ "Meningioma"
-    )
+    ),
+    Notes = "Updated via OpenPBTA subtyping from pathology_free_text_diagnosis"
   ) %>%
   # Drop the column we added for convenience of string detection
-  dplyr::select(-pathology_free_text_dx_lower)
+  # and to format to match compiled_mol_subtypes_pathology_clinical_df
+  dplyr::select(-pathology_free_text_dx_lower,
+                -pathology_free_text_diagnosis)
 ```
 
 Write to file!
+Since Meningioma was not subtyped as part of OpenPBTA molecular subtying we can append the meningioma results to `compiled_mol_subtypes_pathology_clinical_df`.
 
 ```{r}
-readr::write_tsv(meningioma_df, output_file)
+
+meningioma_df %>%
+  bind_rows(compiled_mol_subtypes_pathology_clinical_df) %>%
```

Comment:
```suggestion
  bind_rows(compiled_mol_subtypes_pathology_clinical_df) %>%
  arrange(Kids_First_Biospecimen_ID)
```

Reply:
just going to add this to track future changes, as this file will change a lot as we add the remaining pathology free text updates